store/store.go

```diff
@@ -1098,3 +1110,129 @@ func (s *SnapUbuntuStoreRepository) Buy(options *BuyOptions) (*BuyResult, error)
 		return nil, fmt.Errorf("cannot buy snap %q: unexpected HTTP code %d%s", options.SnapName, resp.StatusCode, details)
 	}
 }
+
+// [
```

Comment:
Hm, it seems its enough to just point to the MockJSON in `store_test.go` instead of having the example here as a comment. This will probably go stale as a comment but we will update the mock json.